Tests/SwiftFusionTests/Core/FixedSizeMatrixTests.swift

```diff
@@ -24,6 +24,18 @@ fileprivate typealias Matrix3x2 = FixedSizeMatrix<Array3<Vector2>>
 fileprivate typealias Matrix3x2_3x3 = FixedSizeMatrix<Array3<Tuple2<Vector2, Vector3>>>
 
 class FixedSizeMatrixTests: XCTestCase {
+  func testVectorConformance() {
+    let s1 = (0..<6).lazy.map { Double($0) }
+    let v1 = Matrix3x2(s1)
+    v1.checkVectorSemantics(expecting: s1, writing: (6..<12).lazy.map { Double($0) })
```

Comment:
I think this would be clearer if it were `checkVectorSemantics(expectingScalars:writingScalars:)`.

Reply:
Done